crates/steel-parser/src/tokens.rs

```diff
@@ -156,9 +156,15 @@ impl From<MaybeBigInt> for BigInt {
     }
 }
 
+#[cfg(test)]
 #[test]
 fn check_token_size() {
```

Comment:
Token size is `64` due to `FractionLiteral` having the capacity for 2 `MaybeBigInt`.

Will look into reducing the size for the rare cases.